README.md

```diff
@@ -25,3 +25,39 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+# Runtime Analysis Answer:
+
+The time complexity of my divideAndConquerSum() is $\Theta(log(n))$.
+At the beginning of my algorithm, I check to see if the length of the
+array is 0 or 1. If the array is either of these lengths, then the execution
+time will be constant in the best case scenario. However, in the scenerio
+where the length is greater than 1, the complexity increases.
+The algorithm will always divide the array up into thirds and process
+its own portion of the array until it reaches the base cases where the 
+sub-array size is 0 or 1, in which it is added in a constant time:
+
+$T(n) = 3T(n/3) + \Theta(1)$
+
+However, we do not necessarily worry about the constant time it takes for 
+the base case to be added, so we will omit it from this analysis:
```

Comment:
No, you can't omit the constant.

Reply:
So in asymptotic analysis, we typically ignore the constant value if there is a large factor such as Theta(logn), Theta(n), Theta(nlogn), etc... But in the context of recurrence relations we we do care about it? I just want to clarify

Reply:
Yes, in general you have to take it into account up until when you get the final asymptotic expression. You can simplify the final complexity that way, but you can't do that before then. Same with lower-order terms etc.